server/github.go

```diff
@@ -128,13 +131,30 @@ func (s *Server) removeLabel(repoOwner, repoName string, number int, label strin
 	mlog.Info("Finished removing the label")
 }
 
-func (s *Server) getComments(repoOwner, repoName string, number int) ([]*github.IssueComment, error) {
-	comments, _, err := s.GithubClient.Issues.ListComments(context.Background(), repoOwner, repoName, number, nil)
-	if err != nil {
-		mlog.Error("pr_error", mlog.Err(err))
-		return nil, err
+func (s *Server) getComments(ctx context.Context, pr *model.PullRequest) (comments []*github.IssueComment, err error) {
+	opts := &github.IssueListCommentsOptions{
+		ListOptions: github.ListOptions{},
```

Comment:
I've been thinking about reducing the number of request and this seams like another good opportunity: By specifying `PerPage` to `100` this only take a third of the requests.

Reply:
:+1: